selfdrive/car/hyundai/values.py

```diff
@@ -1304,6 +1306,19 @@ class Buttons:
       b'\xf1\x00DEE MFC  AT KOR LHD 1.00 1.03 95740-Q4000 180821',
     ],
   },
+  CAR.KIA_NIRO_EV_2ND_GEN: {
+    (Ecu.fwdRadar, 0x7D0, None): [
+      b'\xf1\x00SG2_ RDR -----      1.00 1.01 99110-AT000         ',
+      b'\xf1\x8799110AT000\xf1\x00SG2_ RDR -----      1.00 1.01 99110-AT000         ',
+    ],
+    (Ecu.eps, 0x7D4, None): [
+      b'\xf1\x00SG2 MDPS C A.00 1.03 56310AO000\x00 4SGVC103',
+      b'\xf1\x8756310AO000\x00\xf1\x00SG2 MDPS C A.00 1.03 56310AO000\x00 4SGVC103',
+    ],
```

Comment:
```suggestion
```

Reply:
For my understanding, why are you deleting the "eps" section ?